subprojects/docs/src/docs/dsl/org.gradle.api.plugins.quality.AbstractCodeQualityTask.xml

```diff
@@ -0,0 +1,47 @@
+<section>
+    <section>
+        <title>Properties</title>
+        <table>
+            <thead>
+                <tr>
+                    <td>Name</td>
+                    <td>Default value</td>
+                </tr>
+            </thead>
+            <tr>
+                <td>ignoreFailures</td>
+                <td><literal>project.checkstyle.ignoreFailures</literal></td>
```

Comment:
❌ This is referencing the checkstyle extension, but this applies to all code quality tasks.

🤔 Maybe just put false here?